test_unstructured/partition/pdf_image/test_pdf.py

```diff
@@ -1223,6 +1223,8 @@ def test_partition_pdf_element_extraction(
         if file_mode == "filename":
             elements = pdf.partition_pdf(
                 filename=filename,
+                # Image extraction shouldn't break by setting this
+                starting_page_number=20,
```

Comment:
Could we also add a test that asserts that the resulting page number in `metadata` is correct?